src/app/models/route-option.ts

```diff
@@ -0,0 +1,4 @@
+export interface RouteOption {
```

Comment:
Should be options then - or rather RouteProperties?
This is also missing passengers - or is that in a route?

Reply:
RouteOption, since it will be saved what other option (with different travel modes) exist

Reply:
passengers are missing, fixed in 0c9d589